Assets/BossRoom/Scripts/Game/Client/UI/UnityServicesUIHandler.cs

```diff
@@ -28,22 +28,22 @@ void ServiceErrorHandler(UnityServiceErrorMessage error)
             switch (error.AffectedService)
             {
                 case UnityServiceErrorMessage.Service.Lobby:
-                {
-                    HandleLobbyError(error);
-                    break;
-                }
+                    {
```

Comment:
really? is that in our unity guidelines? That's so ugly!

Reply:
Really? I've always seen code inside a case be indented. Though I can't say the same for braces inside cases. Here, it's just applying the indentation rule.

Reply:
According to our coding convention, braces must be aligned with the “case”
https://drive.google.com/file/d/1JT7jviC31yevd6PnkZUS-amkUFVOoGIQ/view
There’s even an example for it in there.

Reply:
dotnet format seems to still be adding the indent, even when modifying the .editorconfig file to explicitly not do this. Will circle back to this for now, but in the meantime, I've removed the braces.